app/src/main/java/com/kms/cura/model/AppointmentModel.java

```diff
@@ -66,13 +69,16 @@ public static AppointmentModel getInstance() {
         throw new Exception(response.getError());
     }
 
-    public List<AppointmentEntity> updateAppointment(AppointmentEntity appointmentEntity, PatientUserEntity patient) throws Exception {
+    public List<AppointmentEntity> updateAppointment(AppointmentEntity appointmentEntity, UserEntity entity, boolean patient) throws Exception {
```

Comment:
Don't need to pass `boolean patient` to this function. You can check if entity instanceof PatientUserEntity